tests/parsers/cfxmaestro/testdata/example01.json

```diff
@@ -0,0 +1,8467 @@
+                                                "unit": "(unitless)"
+                                            }
+                                        },
+                                        "cycle threshold result": {
+                                            "value": 21.6849883217361,
+                                            "unit": "(unitless)"
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ],
+                    "experiment type": "comparative CT qPCR experiment",
+                    "container type": "well plate"
+                }
+            },
+            {
+                "measurement aggregate document": {
+                    "plate well count": {
+                        "value": 96,
+                        "unit": "#"
+                    },
+                    "measurement document": [
+                        {
+                            "measurement identifier": "CFXMAESTRO_TEST_ID_16",
+                            "measurement time": "1970-01-01T00:00:00+00:00",
+                            "target DNA description": "N/A",
+                            "sample document": {
+                                "sample identifier": "N/A",
+                                "sample role type": "Unkn",
+                                "well location identifier": "A09",
+                                "custom information document": {
+                                    "group identifier": "N/A"
+                                }
+                            },
+                            "device control aggregate document": {
+                                "device control document": [
+                                    {
+                                        "device type": "qPCR",
+                                        "measurement method identifier": "N/A",
+                                        "reporter dye setting": "FAM"
+                                    }
+                                ]
+                            },
+                            "processed data aggregate document": {
+                                "processed data document": [
+                                    {
+                                        "data processing document": {
+                                            "cycle threshold value setting": {
+                                                "value": -0.0,
+                                                "unit": "(unitless)"
+                                            }
+                                        },
+                                        "cycle threshold result": {
+                                            "value": null,
+                                            "unit": "(unitless)"
+                                        }
```

Comment:
@Preston-O-Benchling, what are your thoughts on skipping of wells that contain no `Sample Name` and no `Cq` value? These effectively are empty wells that contain no results data. You have better context if this is enough information to designate whether the well is empty however.

Reply:
Great call! Lets only exclude values if they do not have both a sample name and no Cq value. This way if the user accidentally forgets to label a well or mixes up the well plate, the value wont be excluded.

Reply:
Done!